wiki/markdown/configuration.md

```diff
@@ -1,39 +1,43 @@
 # Configuration
 
-Address and port for message buss and MongoDB needs to be configured in 
-[application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties) 
-for the application to run properly.
-
-Each property is explained in the file so open the properties file for a view 
+Address and port for message buss and MongoDB needs to be configured in
```

Comment:
message "buss" should be spelled "bus".

Reply:
This can be rewritten so it don't say that it only can configure MessageBus and MongoDb, so description covers all settings:
"MessageBus, MongoDb and other Eiffel-Intelligence settings is configured in applications.properties: .... "

Reply:
Ok, will fix